src/main/java/com/blog/controller/PostsController.java

```diff
@@ -76,6 +76,44 @@ public String getPagePost(@PathVariable(name = "id") Long id, Model model) {
         return "blogPost";
     }
 
+    @GetMapping("/{postId}/update")
+    public String getPageForUpdatePost(
+            Model model,
+            @PathVariable(value = "postId") Long postId) {
+        List<Tag> tags = tagService.getAllTags();
+
+        ResponsePostDto post = postDao.getPostById(postId);
+        Map<String, String> currentPage = pageService.getPageDefaultParams();
+
+        currentPage.put("title", "update");
+
+        setActiveUserInModelAttribute(model);
```

Comment:
method has side effect.
It's ugly

Reply:
I agree with you, but this is a temporary solution until the security is implemented.

Reply:
This is necessary so that when implementing security it is not necessary to completely redo the front end.

Reply:
1. Are you sure that you will have the same method signature? Think no, as a result you still have to refactor code.
2. I don't have any problems with the main goal of this method. My point is why does this method change state of `model` instead of returning current user?

Reply:
I'm sorry, I understood you wrong, I'll fix it.